code/datums/traits/good.dm

```diff
@@ -41,6 +41,22 @@
 	mob_trait = TRAIT_EMPATH
 	gain_text = "<span class='notice'>You feel in tune with those around you.</span>"
 	lose_text = "<span class='danger'>You feel isolated from others.</span>"
+	
+/datum/quirk/Supportanimal
```

Comment:
Please make this name follow naming scheme in this file

Reply:
in snake case support_animal